sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallAutomationClientBuilder.java

```diff
@@ -81,6 +82,7 @@ public final class CallAutomationClientBuilder
     private RetryPolicy retryPolicy;
     private RetryOptions retryOptions;
     private CommunicationUserIdentifier sourceIdentity;
+    private MicrosoftTeamsAppIdentifier oPSSourceIdentity;
```

Comment:
nit: oPS look a bit weird here, as OPS does stand for OnePhoneSystem.
We should try to do all capital (OPS..) or all lower (ops..) or full name (onePhoneSystem...) 
Maybe we can consider make acronym small letter? (i.e. opsSourceIdentity)